hack/cr.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: "elasticsearch"
 spec:
   nodeSpec:
-    image: openshift/origin-logging-elasticsearch5:latest
+    image: 172.30.1.1:5000/openshift/logging-elasticsearch5:latest
```

Comment:
why is this necessary?  It seems to work without it, in my testing

Reply:
Assuming we build and push the image with this to dockerhub this change can be omitted.  I will investigate

Reply:
a test deploy or test-e2e should not push an image to an external registry
are you saying that if you build a local image, you must push it to the local registry in order for it to be used?  For example
```
cd origin-aggregated-logging
docker build -t openshift/origin-logging-elasticsearch5:latest -f elasticsearch/Dockerfile.centos7 elasticsearch/
```
This creates a local image `openshift/origin-logging-elasticsearch5:latest` - `docker images` shows this image

Are you saying that if I then deploy elasticsearch, it will attempt to pull `openshift/origin-logging-elasticsearch5:latest` from `hub.docker.io` or `quay.io`, even though the image is available locally?